components/Ink/tests/unit/MDCInkLayerTests.m

```diff
@@ -146,17 +146,15 @@ - (void)testEndAnimationTimingInTimeScaledLayer {
   inkLayer.endAnimationDelay = (CGFloat)0.9;
 
   // When
-  NSTimeInterval startTime = CACurrentMediaTime();
   [inkLayer endAnimationAtPoint:CGPointMake(5, 5)];
-
+  NSTimeInterval startTime = CACurrentMediaTime();
 
   // Then
   XCTAssertEqual(inkLayer.addedAnimations.count, 1U);
   CAAnimation *animation = inkLayer.addedAnimations.firstObject;
   if (animation) {
-    XCTAssertEqualWithAccuracy(animation.beginTime,
-                               [inkLayer convertTime:(startTime + 0.9) fromLayer:nil],
-                               0.005);
+    startTime = [inkLayer convertTime:(startTime + 0.9) fromLayer:nil];
+    XCTAssertEqualWithAccuracy(animation.beginTime, startTime, 0.010);
```

Comment:
Any reason not to go to 0.10? Given the speed (0.5) and the duration (0.9), I think being within 10 ms should be sufficient to detect whether or not the scale is off by 450 ms.